pkg/tarmak/cluster/cluster.go

```diff
@@ -428,6 +428,8 @@ func (c *Cluster) Variables() map[string]interface{} {
 		ids, ok := imageIDs[image]
 		if ok {
 			output[fmt.Sprintf("%s_ami", instancePool.TFName())] = ids
+		} else {
+			c.log.Fatalf("error getting the image ID of %s", instancePool.TFName())
```

Comment:
I think this needs to happen at another stage before plan/apply commands. We have to be sure that this method is not producing any errors. Furthermore having Fatal calls anywhere in pkg/ should be avoided at any costs